bigchaindb/models.py

```diff
@@ -208,8 +208,8 @@ def validate(self, bigchain):
         """Validate the Block.
 
         Args:
-            bigchain (:class:`~bigchaindb.Bigchain`): An instantiated Bigchain
-                object.
+            bigchain (:class:`~bigchaindb.tendermint.BigchainDB`): An
+                instantiated BigchainDB object.
```

Comment:
This whole class is going to be removed in https://github.com/bigchaindb/bigchaindb/pull/2368